<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChaCha20</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Enkripsi</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="9,1">
       <property name="spacing">
        <number>65</number>
       </property>
       <property name="topMargin">
        <number>50</number>
       </property>
       <property name="bottomMargin">
        <number>135</number>
       </property>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>200</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="lineWidth">
             <number>2</number>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="../resource.qrc">:/images/images/lock.svg</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
            <property name="margin">
             <number>15</number>
            </property>
            <property name="indent">
             <number>-1</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,3,1">
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Enkripsi</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Dekripsi</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="9,1">
       <property name="spacing">
        <number>65</number>
       </property>
       <property name="topMargin">
        <number>50</number>
       </property>
       <property name="bottomMargin">
        <number>135</number>
       </property>
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>200</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="lineWidth">
             <number>2</number>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="../resource.qrc">:/images/images/unlock.svg</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
            <property name="margin">
             <number>18</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,3,1">
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Dekripsi</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>777</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionPlaintext"/>
    <addaction name="actionCipherText"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuBantuan">
    <property name="title">
     <string>Bantuan</string>
    </property>
    <addaction name="actionSpek"/>
    <addaction name="actionPenggunaan"/>
    <addaction name="actionTentang"/>
   </widget>
   <widget class="QMenu" name="menuLihat">
    <property name="title">
     <string>Lihat</string>
    </property>
    <addaction name="actionPerkecil"/>
    <addaction name="actionPerbesar"/>
   </widget>
   <widget class="QMenu" name="menuTest_Vector">
    <property name="title">
     <string>Fitur</string>
    </property>
    <addaction name="actionTest_Vector"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuLihat"/>
   <addaction name="menuTest_Vector"/>
   <addaction name="menuBantuan"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionPlaintext">
   <property name="text">
    <string>PlainText</string>
   </property>
  </action>
  <action name="actionCipherText">
   <property name="text">
    <string>CipherText</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionPenggunaan">
   <property name="text">
    <string>Penggunaan</string>
   </property>
  </action>
  <action name="actionTentang">
   <property name="text">
    <string>Tentang</string>
   </property>
  </action>
  <action name="actionPerkecil">
   <property name="text">
    <string>Perkecil Jendela</string>
   </property>
  </action>
  <action name="actionPerbesar">
   <property name="text">
    <string>Perbesar Jendela</string>
   </property>
  </action>
  <action name="actionSpek">
   <property name="text">
    <string>Spesifikasi</string>
   </property>
  </action>
  <action name="actionTest_Vector">
   <property name="text">
    <string>Uji Vektor</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resource.qrc"/>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
